extension/scripts/content/hospital/ttHospital.js

```diff
@@ -4,7 +4,7 @@
 	document.addEventListener("click", (event) => {
 		if (event.target.classList.contains("page-number") || event.target.classList.contains("page-nb")) {
 			requireElement(".user-info-list-wrap .last #iconTray li").then(() => {
-				triggerCustomListener(EVENT_CHANNELS.HOSPITAL_SWITCH_PAGE, null);
+				triggerCustomListener(EVENT_CHANNELS.HOSPITAL_SWITCH_PAGE, "pageChange");

```

Comment:
The event already states that it's a page change? Otherwise, it should be an object.

Reply:
An empty object isn't the way to go ....